geopandas/base.py

```diff
@@ -143,13 +183,68 @@ def type(self):
 
     @property
     def length(self):
-        """Returns a ``Series`` containing the length of each geometry."""
+        """Returns a ``Series`` containing the length of each geometry.
+
+        Examples
+        --------
+
+        >>> from shapely.geometry import Polygon
+        >>> s = geopandas.GeoSeries(
+        ...     [
+        ...         Polygon([(0, 0), (1, 1), (0, 1)]),
+        ...         Polygon([(10, 0), (10, 5), (0, 0)]),
```

Comment:
Suggestion: it would be good focus this more on geometry types where length is more applicable, and secondarily demonstrate it for polygons (and also show that it is 0 for points): LineString, MultiLineString, Polygon, Point. (maybe also a GeometryCollection fo mixed types?)